src/mailing/sender.js

```diff
@@ -76,11 +76,18 @@ async function sendWelcomeEmail(recipient, user) {
 
 async function sendForgotPasswordEmail(toEmail, resetToken) {
   try {
+    const frontendBaseUrl =
+      process.env.NODE_ENV === 'production'
+        ? process.env.FRONTEND_BASE_URL_PROD
+        : process.env.FRONTEND_BASE_URL_DEV;
```

Comment:
We can even just make this:

```suggestion
    const frontendBaseUrl = process.env.FRONTEND_BASE_URL
```

In dev mode, we have a `.env` file where we set this value. We would usually have this value set to the localhost one. But if for some reason a dev really wanted to test against the prod url they would just change the value in the .env file, without havign to fully change the `NODE_ENV` to production.

In production, we set this value in [https://render.com/](https://render.com/) dashboard (make sure to do that before merging). There's no need to have separate env values for prod and dev, because the places where we set these values are already so separate.